src/common/components/ui/formik/RemovableField.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { Button, ButtonVariant, Tooltip, TooltipProps } from '@patternfly/react-core';
+import { MinusCircleIcon } from '@patternfly/react-icons';
+import { FieldArrayRenderProps } from 'formik';
+import './RemovableField.css';
+
+interface RemovableFieldProps {
+  index: number;
+  remove: FieldArrayRenderProps['remove'];
```

Comment:
similar to `AddButton`, lets just have `onRemove: VoidFunction`
```suggestion
  onRemove: VoidFunction
```